docs/api-examples/lob.mdx

```diff
@@ -4,7 +4,7 @@ In this example, we will show you how you can get started working with the [Lob
 
 :::info Before you start
 
-You will need a Lob API key. Find out how to set that up in the [API Keys](https://docs.lob.com/#section/API-Keys) section of Lob's Authentication documentation.
+You will need a Lob API key. Find out how to set that up in the [API Keys](https://dashboard.lob.com/settings/api-keys) section of Lob's Authentication documentation.
```

Comment:
note: The link no longer points at Lob's docs, but Accnt Settings

Reply:
Yes. Changed!